oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -545,30 +545,23 @@ private APIGatewayProxyResponseEvent generateAuthRedirect(
             List<VectorOfTrust> vtrList,
             TxmaAuditUser user) {
         LOG.info("Redirecting");
-        String redirectURI;
-        try {
-            var redirectUriBuilder = new URIBuilder(configurationService.getLoginURI());
-
-            redirectUriBuilder.setPath("authorize");
-
-            if (Objects.nonNull(authenticationRequest.getPrompt())
-                    && authenticationRequest.getPrompt().contains(Prompt.Type.LOGIN)) {
-                redirectUriBuilder.addParameter("prompt", String.valueOf(Prompt.Type.LOGIN));
-            }
 
-            List<String> optionalGaTrackingParameter =
-                    authenticationRequest.getCustomParameter(GOOGLE_ANALYTICS_QUERY_PARAMETER_KEY);
-            if (Objects.nonNull(optionalGaTrackingParameter)
-                    && !optionalGaTrackingParameter.isEmpty()) {
-                redirectUriBuilder.addParameter(
-                        GOOGLE_ANALYTICS_QUERY_PARAMETER_KEY, optionalGaTrackingParameter.get(0));
-            }
+        Optional<Prompt.Type> prompt = Optional.empty();
+        if (Objects.nonNull(authenticationRequest.getPrompt())
+                && authenticationRequest.getPrompt().contains(Prompt.Type.LOGIN)) {
+            prompt = Optional.of(Prompt.Type.LOGIN);
+        }
```

Comment:
How do you feel about this instead?

```suggestion
        var prompt =
                Objects.nonNull(authenticationRequest.getPrompt())
                                && authenticationRequest.getPrompt().contains(Prompt.Type.LOGIN)
                        ? Optional.of(Prompt.Type.LOGIN)
                        : Optional.empty();
```

Reply:
Looks good